.github/workflows/test.yaml

```diff
@@ -39,6 +39,7 @@ jobs:
     - name: Get Changed Files
       id: get_changed_files
       uses: jitterbit/get-changed-files@v1
+      continue-on-error: true
```

Comment:
This is how c551efc was tested out:

1. I made a new branch off of `main` with a new test markdown file. The file doesn't matter; what matters is that the branch was made off of `main`. I then made a pull request that targeted `develop` (not `main`) for this branch:

    - https://github.com/linode/docs/pull/4306
    - Vale failed on this with the `head commit` error because `main`, which was used to start the new branch, is currently behind `develop`

1. I made another new branch off of `main` with the proposed `continue-on-error: true` fix. Then I made a pull request that targeted `develop` (not `main`) for this branch:

    - https://github.com/linode/docs/pull/4305
    - Vale passes on this PR, even though `main`, which was used to start the new branch, is currently behind `develop`.